test/functional/wallet_upgradewallet.py

```diff
@@ -333,8 +332,8 @@ def copy_split_hd():
         # Check the wallet has a default key initially
         old_kvs = dump_bdb_kv(node_master_wallet)
         defaultkey = old_kvs[b'\x0adefaultkey']
-        # Upgrade the wallet. Should still have the same default key
-        wallet.upgradewallet(159900)
+        self.log.info("Upgrade the wallet. Should still have the same default key.")
+        self.test_upgradewallet(wallet, previous_version=139900, requested_version=159900)
```

Comment:
In commit 4dc34408bfe8fa1c229d99a3d0d03a42c83ced69 "wallet: fix and improve upgradewallet result responses"

The `assert_equal(wallet.getwalletinfo()["walletversion"], 159900)` below can be removed now.

Reply:
Thanks for confirming that. Done.